recipe/meta.yaml

```
@@ -10,7 +10,10 @@ source:
   sha256: 1c211b944ac83ef013fe91aee96c01289da4e3762c1e2f265449f3a964f8e4bc
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    # This is just a guess
+    - {{ pin_subpackage('svt-av1', max_pin='x.x') }}
```

I think @matthiasdiener should ask for clarification from upstream developers about [their ABI compatibility guarantees](https://github.com/AOMediaCodec/SVT-AV1/blob/6e87a1de98281840abebc030781780edd822bae5/CHANGELOG.md?plain=1#L14). If they are completely independent, then the pinning may need to be exact/all the way to minor version.
```suggestion
    - {{ pin_subpackage('svt-av1', max_pin='x.x.x') }}
```

---

Do you want to pin hard and loosen later?

---

Yeah, let's pin it more strictly for now I think.